src/components/scenes/courses/CourseManagement/components/GuideManagement/GuideManagement.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import { Container, Typography } from '@material-ui/core';
+import Fab from '@material-ui/core/Fab';
+import AddIcon from '@material-ui/icons/Add';
+import Grid from '@material-ui/core/Grid';
+import ExerciseManagement from '../ExerciseManagement/ExerciseManagement';
+import styles from '../ExerciseManagement/ExerciseManagement.module.sass';
+
+export default class GuideManagement extends Component {
+  componentDidMount() {
+    this.updateExercises();
+  }
+
+  componentDidUpdate(prevProps) {
+    const { location } = this.props;
+    if (location !== prevProps.location) {
```

Comment:
I don't understand this

Reply:
When a guide is selected. If the guide is different from the previous (the location is different), then fetch the exercises.